gateway/Cargo.toml

```diff
@@ -37,6 +37,13 @@ metrics = { default-features = false, features = ["std"], optional = true, versi
 simd-json = { default-features = false, features = ["serde_impl", "swar-number-parsing"], optional = true, version = "0.4" }
 tracing = { default-features = false, features = ["std", "attributes"], optional = true, version = "0.1" }
 
+# TLS libraries
+# They are needed to track what is used in tokio-tungstenite
+native-tls = { default-features = false, features = [], optional = true, version = "0.2.8" }
+rustls-tls = { default-features = false, features = [], optional = true, package = "rustls", version = "0.20" }
+rustls-native-certs = {default-features = false, features = [], optional = true, version = "0.6.1" }
```

Comment:
also question: what is in 0.6.1 that we need which is not in 0.6.0? seems like the features added are irrelevant here